README.md

```diff
@@ -116,4 +116,6 @@ string_value:"5"
 
 * Not all models support dynamic batch sizes.
 
-* The backend needs to be loaded in Triton in persistent mode, otherwise it might lead to segmentation fault while unloading.
\ No newline at end of file
+* The backend needs to be loaded in Triton in persistent mode, otherwise it might lead to segmentation fault while unloading.
+
+* Openvino does not natively support dynamic/variable shape tensors. If the outermost dimension is the only variable dimension (for batch models the outermost dimension is the batch dimension), we can set this dimension to a large value and enable ENABLE_BATCH_PADDING in the backend configuration as a work around. However, if any other dimension is dynamic i.e. -1, the model will not be supported in Openvino and hence is not supported by the Openvino backend in Triton.
```

Comment:
There is a way to support variable shape in OV. Hence. I would rephrase the issue like below:
"As of now backend does not support variable shaped tensors. However, the dynamic batch sizes in the model are supported. See `SKIP_OV_DYNAMIC_BATCHSIZE` and `ENABLE_BATCH_PADDING` parameters for more details."

Reply:
Also, the first bullet about persistent mode is not something the user can control or needs to be aware of.

Reply:
Fixing the issue with tanmay's suggestion and remove the earlier statement about persistent mode